<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_new_soi</class>
 <widget class="QDialog" name="Dialog_new_soi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New SOI</string>
  </property>
  <widget class="QFrame" name="frame2_1">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>370</width>
     <height>290</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton_ok">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>257</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cancel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>257</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label2_149">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>75</y>
      <width>150</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>End Frequency (MHz):</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label2_146">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>105</y>
      <width>150</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Bandwidth (MHz):</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label2_147">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>165</y>
      <width>150</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Continuous:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_continuous">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>200</width>
      <height>27</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <item>
     <property name="text">
      <string>True</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>False</string>
     </property>
    </item>
   </widget>
   <widget class="QTextEdit" name="textEdit_bandwidth">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>200</width>
      <height>27</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label2_148">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>45</y>
      <width>150</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start Frequency (MHz):</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_modulation">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>200</width>
      <height>27</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_end_frequency">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>200</width>
      <height>27</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label2_145">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>135</y>
      <width>150</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Modulation:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_start_frequency">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>200</width>
      <height>27</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_frequency">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>200</width>
      <height>27</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label2_142">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>15</y>
      <width>150</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Center Frequency (MHz):</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label2_150">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>195</y>
      <width>150</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Notes:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_notes">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>200</width>
      <height>61</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
